packages/react-core/src/components/Pagination/Navigation.tsx

```diff
@@ -201,16 +204,20 @@ export class Navigation extends React.Component<NavigationProps, NavigationState
               className={css(styles.formControl)}
               aria-label={currPage}
               type="number"
-              disabled={isDisabled || (page === firstPage && page === lastPage) || page === 0}
+              disabled={
+                isDisabled || (itemCount && page === firstPage && page === lastPage && itemCount >= 0) || page === 0
+              }
               min={lastPage <= 0 && firstPage <= 0 ? 0 : 1}
               max={lastPage}
               value={userInputPage}
               onKeyDown={event => this.onKeyDown(event, page, lastPage, onPageInput)}
               onChange={event => this.onChange(event, lastPage)}
             />
-            <span aria-hidden="true">
-              {ofWord} {pagesTitle ? pluralize(lastPage, pagesTitle) : lastPage}
-            </span>
+            {itemCount && (
```

Comment:
`itemCount` needs to be updated to `(itemCount || itemCount === 0)` for the case of 0 items, rather than unknown items.

Reply:
Oh good catch! :)